packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -366,6 +370,33 @@ export class InMemoryAnchorService implements AnchorService, AnchorValidator {
     return anchorCid
   }
 
+  /**
+   * Builds the CAR file that the AnchorService responds to Ceramic with for a successfully anchored
+   * request. Contains the AnchorCommit as well as the merkle tree path to the anchored commit.
+   * For the InMemoryAnchorService, the tree is always of size one, so there are only 3 objects in
+   * the CAR file: the AnchorCommit, the AnchorProof for the batch, and the commit that was anchored.
+   */
+  async _buildWitnessCAR(proofCid: CID, anchorCommitCid: CID, updateCid: CID): Promise<CAR> {
+    const car = this.carFactory.build()
+
+    const cidToBlock = async (cid) => new CarBlock(cid, await this.#dispatcher.getIpfsBlock(cid))
+
+    car.blocks.put(await cidToBlock(proofCid))
+    car.blocks.put(await cidToBlock(anchorCommitCid))
+    car.blocks.put(await cidToBlock(updateCid))
```

Comment:
it's not strictly required for the CAR file to include the update that was anchored, as this node should already have that data since it was the one who requested that update be anchored in the first place.

I'd like to mirror whatever the CAS will do here - the leaf nodes of the anchor merkle tree are the updates that were anchored in the batch, so we'd probably have to be careful when constructing the CAR files in the CAS to exclude the updates if we want to save the storage and bandwidth costs from not including them.

Reply:
Removed the update from the CAR file